### DISCUSSION TITLE
Can I use a new account since my main account is banned

### DISCUSSION THREAD

**Discussion by ghost** [community/community#134903] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My main account got randomly suspended with no reason nor notice. I send an appeal but have got nothing and it has been over a month. Other people I have seen said they had to wait 100+ days. I do not have that kind of time. 

I am wondering if using an a different account for the time being is okay. Will it get banned eventually? Technically, my banned account doesn't exist anymore. This is what the TOS says:
```
One person or legal entity may maintain no more than one free Account (if you choose to control a machine account as well, that's fine, but it can only be used for running a machine).
```
Don't really understand what the parathesis mean.

Most people on [here](https://www.reddit.com/r/github/comments/14w872v/am_i_allowed_to_have_multiple_accounts_on_github/) are saying that it is just difficult to manage multiple accounts. However, I am only managing one, since the other is banned.

**Comment by LiteBrite82** [comment]
Hi @Pymon02 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by ghost** [reply]
That was not my question.
